Crashlytics/UnitTests/Mocks/FIRMockInstallations.m

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 Google
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "FIRMockInstallations.h"
+
+#import <Foundation/Foundation.h>
+
+//#import <FirebaseInstanceID/FIRInstanceID_Private.h>
+
+@interface FIRMockInstallationsImpl : NSObject
+
+@property(nonatomic, copy) NSString *installationID;
+@property(nonatomic, strong) NSError *error;
+
+@end
+
+@implementation FIRMockInstallationsImpl
+
+- (void)installationIDWithCompletion:(FIRInstallationsIDHandler)completion {
+  completion(self.installationID, self.error);
+}
+
+@end
+
+@interface FIRMockInstallations ()
```

Comment:
We don't need the empty category with the parenthesis ()?

Reply:
This is a [class extension](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/CustomizingExistingClasses/CustomizingExistingClasses.html#//apple_ref/doc/uid/TP40011210-CH6-SW3) declaration needed here to declare the private properties. BTW, this class is just slightly modified version of the old [FIRMockInstanceID.m](https://github.com/firebase/firebase-ios-sdk/pull/4954/files/cb78d8099f954ef4682af9ddac13439e0c401135#diff-e0d6eba2cde9465269c3140fb0507a05), GitHub doesn't show it properly for some reason (probably because of the file renaming).